docs/tutorial/kymotracking.rst

```diff
@@ -0,0 +1,206 @@
+.. warning::
+    This is early access alpha functionality. While usable, this has not yet been tested in a large number of different
+    scenarios. The API is still be subject to change *without any prior deprecation notice*! If you use this
+    functionality keep a close eye on the changelog for any changes that may affect your analysis.
+
+Kymotracking
+============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+
+Kymographs are a vital tool in understanding binding events on DNA. They are generated using one-dimensional scans
+over time. Moving particles can then be seen moving along the spatial coordinate in time. It is often desired to
+to track the position of such particles in time. Pylake comes with a few features that enable you to do so.
+
+
+Performing the tracking
+-----------------------
+
+Performing Kymotracking is usually performed in two steps. Typically, there is an image processing step, following by
+a tracing step, where the actual traces are found.
+
+
+Using the greedy algorithm
+--------------------------
+
+First, we need to get an image to perform the tracing on. Let's grab a kymograph from a Bluelake file::
+
+    file = lk.File('kymograph.h5')
+    _, kymo = file.kymos.popitem()
+
+We can extract the image data with one of the `_image` commands::
+
+    data = kymo.green_image
+
+Let's have a look at what it looks like::
+
+    plt.imshow(data, aspect="auto", vmax=5)
+
+.. image:: kymo_unscaled.png
+
+What we also see is that we can clearly see the bead edges in the kymograph. Let's only take a subsection of the
+kymograph, to make sure that we don't get any spurious line detections that come from the bead edges. We only use the
+70th to the 220th pixel.
+
+    >>> data = data[70:220, :]
+
+If we zoom in a bit, we can see that our traces in this image are about 3 pixels wide, so let's set this as our
+`line_width`.
+
+.. image:: kymo_zoom.png
+
+The peaks have an intensity of about 3-7 photon counts, whereas the background fluctuates around 0-2. Let's set our
```

Comment:
Consider adding a colorbar to the plot to make this more obvious in the docs without interactive widgets.